### DISCUSSION TITLE
How is the website built after every commit in a pull request

### DISCUSSION THREAD

**Discussion by san-coding** [layer5io/layer5#2240] [Q&A] [answered]
How is the website built after every commit in a  pull request , I added a community contributor profile and I could see a live hosted link to view my changes after every commit to that PR, please explain how to do that or how it works @leecalcote @warunicorn19 @Zain-Muiz 

This is what I am referring to :

https://github.com/layer5io/layer5/pull/2208#issuecomment-923875631

**Comment by leecalcote** [comment, marked as answer]
This has been answered a couple times now. Most communities and projects do not openly and patiently teach contributors as they are attempting to work on the community's projects. We do, though! We spend an inordinate amount of time teaching people - - I mean a LOT of time. It's costly. Randomly teaching people how to do things for their own projects and their own purposes... well, it's a burden we can't carry. 

Please see https://discuss.layer5.io for further discussion on Layer5 projects, if you would, as we're transitioning from GH discussions to that forum.
